sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkIntegrationTest.java

```diff
@@ -533,10 +533,10 @@ public void shouldPropagateMetadataWithHttpDeferredCall() {
     Assertions.assertEquals(value, actionResponse);
   }
 
-  @Ignore
+  @Test
   public void searchWithInstant() {
 
-    var now = Instant.now();
+    var now = Instant.now().truncatedTo(SECONDS);
```

Comment:
millis/macros would be just fine, but it doesn't matter

Reply:
I feel safer with seconds. 🤣